libraries/chain/include/eosio/chain/snapshot.hpp

```diff
@@ -5,6 +5,7 @@
 #include <fc/variant_object.hpp>
 #include <boost/core/demangle.hpp>
 #include <ostream>
+#include "../rapidjson/include/rapidjson/document.h"
```

Comment:
it really should be added to cmake as an INTERFACE library and properly set up so only need to `#include <rapidjson/document.h>`